src/include/sof/trace.h

```diff
@@ -35,30 +35,35 @@
  * when main trace is not available
  */
 #define TRACE_BOOT_START		0x1000
-#define TRACE_BOOT_ARCH		0x2000
+#define TRACE_BOOT_ARCH			0x2000
 #define TRACE_BOOT_SYS			0x3000
```

Comment:
You are fixing indentation here, right? Please make it a separate patch.

Reply:
Done.